src/dev/konnector.md

```diff
@@ -2,110 +2,376 @@
 
-A connector is a simple script which imports data from other web services and put it in your cozy.
-Each connector is an independant application, managed by the [Cozy Collect]
-application.
+A connector (as know as _konnector_) is a script that imports data from another web service and put those data into your cozy.
+Each connector is an independant application, managed by the [Cozy Collect] application.
 
-To protect your data, each connector runs inside a container in order to sandbox all their
-interactions with your data.
+To protect your data, each connector runs inside a container in order to sandbox all their interactions with your data.
 
-## How does it work ?
+## How does it work?
 
-A connector is a NodeJS script. The target node version used to run your connector is the
-[current LTS version](https://nodejs.org) (8 at the moment).
+A connector is a NodeJS script.
+The target node version used to run your connector is the [current LTS version](https://github.com/nodejs/Release#release-schedule) (8 at the time this doc was writed).
 
-Like client side apps, connectors communicate with the [Cozy Stack]
-using its API, and get an auth token every time they start. They need to register with a manifest,
-and ask permissions to the user.
+Like client side apps, connectors communicate with the [Cozy Stack] using its HTTP API, and get an unique auth token every time they start.
+They need to register with a manifest, and ask the user for permissions.
 
-To ease the development of a connector, a npm package, named [cozy-konnector-libs]
-provides some shared libraries which are adapted to be used for a connector :
+To facilitate connector development, a npm package, [konnectors/libs], provides some shared libraries that are adapted to be used for a connector:
 
- - [cheerio](https://cheerio.js.org) to easily request html pages like jQuery
- - [request-promise](https://github.com/request/request-promise): [request](https://github.com/request/request) wrapped in promises
- - [request-debug](https://github.com/request/request-debug) which displays all the requests and
-   responses in the standard output. Check "debug" option in [requestFactory](https://github.com/cozy/cozy-konnector-libs/blob/master/packages/cozy-konnector-libs/docs/api.md#module_requestFactory)
+ - [cheerio](https://cheerio.js.org) to easily query a DOM
+ - [request-promise](https://github.com/request/request-promise): [request](https://github.com/request/request) with Promise support
+ - [request-debug](https://github.com/request/request-debug) that displays all the requests and responses in the standard output.
+   Toggle _debug_ option in [requestFactory options](https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/docs/api.md#module_requestFactory)
 
-But you may need some other npm packages not integrated in [cozy-konnector-libs] to help you run your connector :
+Besides, you'll probably need some other npm packages to help you run your connector:
```

Comment:
Besides c'est "au fait", "au passage". Si on veux dire "En plus", peut-être `Aside from that`?

Reply:
Je pensais plus à quelque chose comme ça :

> En outre, vous aurez probablement besoin d'autres paquets npm pour vous aider à faire fonctionner votre connecteur :